### ISSUE TITLE
Users should not be able to edit the journal entries after they have been submitted for review or posted to the journal

### ISSUE THREAD

**Issue by maratmuginov** [maratmuginov/Pacioli#3]

**Comment by itiannerona** [comment]
You mentioned users, how many people can work on a journal? If a journal has been reviewed, can it not be sent back to the user(s) for revisions? Also, can these users be represented as Accountant entity?

**Comment by maratmuginov** [comment]
> You mentioned users, how many people can work on a journal? If a journal has been reviewed, can it not be sent back to the user(s) for revisions? Also, can these users be represented as Accountant entity?

Yes I believe one person should work on a journal entry at a time. That means only one person can create it and one (or more) different people can review it without being able to edit it. Perhaps we can have an attribute on the Journal entry that describes the user who originated it? 

Only Users with the Accountant role should be able to make entries and approve entries to the Journal.
